datafusion/sql/src/unparser/ast.rs

```diff
@@ -242,8 +242,8 @@ impl SelectBuilder {
             from: self
                 .from
                 .iter()
-                .map(|b| b.build())
-                .collect::<Result<Vec<_>, BuilderError>>()?,
+                .filter_map(|b| b.build().ok().unwrap())
```

Comment:
I think this change will result in a panic on a `UninitializedFieldError` due to the `unwrap`. E.g. if self.relation is `None` rather than `Some(None)`. I think we could avoid this by using [Result::transpose](https://doc.rust-lang.org/std/result/enum.Result.html#method.transpose) rather than `.ok().unwrap()` here and then continuing to `collect::<Result<T>, _>`.

Reply:
Thanks, @devinjdangelo. Nice idea. I've learned more about Rust.